vignettes/blueprint/_setup.Rmd

```diff
@@ -6,5 +6,5 @@ pre.mermaid {
 ```
 
 ```{r, echo=FALSE}
-shiny::tags$script(type = "module", "import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid/+esm'")
+tags$script(type = "module", "import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid/+esm'")
```

Comment:
Examples and vignettes should attach packages. But `teal` `Depeneds` on `shiny` so `library(teal)` should suffice.

Reply:
totally! +1 for `library(teal)`

Reply:
[3ebd0e2](https://github.com/insightsengineering/teal/pull/1122/commits/3ebd0e2ab4d7b7e0472d68e484a7e3e513bb3530)

Reply:
On my initial comment I didn't notice this was on the `_setup.Rmd` file so I incorrectly gave the `library(shiny)` suggestion.

Reply:
Using shiny here to avoid having html code, but if you prefer we could replace the whole chunk with:

```html
<script type="module">import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid/+esm'</script>
```

Reply:
> Guys, this is a trick to load a javascript library on all of the blueprint's pages. I'd argue heavily against adding a library call here as this will load the package in all of Blueprint's pages.

This should have been raised when the blueprint was introduced. If this is indeed an issue, please create a separate ticket as this is likely to precipitate a discussion and is out of scope.

Reply:
This was discussed when blueprint was introduced https://github.com/insightsengineering/coredev-tasks/issues/467#issuecomment-1731705180

Reply:
> adding a new heavy dependency to the package that doesn't even support

I did not get this part @averissimo?  we added `library(teal)`

Reply:
I meant that adding the `{DiagrammeR}` R package to `{teal}` that would allow for the mermaid diagrams to be plotted in Rmd would add a new heavy Suggests depedency to teal.